tests/test_transforms/test_math/test_limit_transform.py

```diff
@@ -0,0 +1,146 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from etna.datasets import TSDataset
+from etna.metrics import MSE
+from etna.models import NaiveModel
+from etna.pipeline import Pipeline
+from etna.transforms.math import LimitTransform
+
+
+@pytest.fixture
+def ts_check_pipeline_with_limit_transform(random_seed) -> TSDataset:
+    periods = 100
+    df1 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df1["segment"] = ["segment_1"] * periods
+    df1["target"] = np.arange(periods)
+    df1.loc[5, "target"] = np.NaN
+
+    df2 = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df2["segment"] = ["segment_2"] * periods
+    df2["target"] = np.random.uniform(-15, 5, size=periods)
+    df2.loc[10, "target"] = np.NaN
+
+    df = pd.concat((df1, df2))
+    df = TSDataset.to_dataset(df)
+    tsds = TSDataset(df, freq="D")
+    return tsds
+
+
+@pytest.fixture
+def ts_check_limit_transform(random_seed) -> TSDataset:
+    periods = 4
+    df = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df["segment"] = ["segment_1"] * periods
+    df["target"] = np.array([1, 2, 3, 4])
+    df["target_transformed_1"] = np.array([1, 2, 3, 4])
```

Comment:
Better to create separate fixture with the answers
1. ts_check_limit_transform = target, non_target
2. ts_check_limit_transform_expected =  target, non_target + answers

Then in tests you do the following
```
transform.fit_transform(ts)
df = ts.to_pandas()
df_expected = ts_expected.to_pandas(features=["target", "non_target", "n_column])
```

Reply:
I can't understand why do we need one extra df because in your example `ts_check_limit_transform_expected` is a copy of `ts_check_limit_transform` but plus one new column

Reply:
The idea is that transform works with the whole dataset and can change not only the in_column  -- this way we preserve the values of the other columns

Reply:
But here it may be really hard to come up with good test case